### PR TITLE
Card was created

### PR DESCRIPTION
**Pull request by NazariiAlieksieiev**
- [DEMO LINK](https://NazariiAlieksieiev.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://NazariiAlieksieiev.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,63 @@
...
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    
+    <div class="product-card" data-qa="card">
```
Comment (AlisaVlasova) [inline src/index.html:15]:
```suggestion
    <div class="card" data-qa="card">
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,63 @@
...
+      </p>
+
+      <div class="rating">
+        <div class="rating__stars">
+          <div class="rating__star"></div>
+          <div class="rating__star"></div>
+          <div class="rating__star"></div>
+          <div class="rating__star"></div>
+          <div class="rating__star"></div>
+        </div>
```
Comment (AlisaVlasova) [inline src/index.html:37]:
```suggestion
      <div class="card__rating">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```

Comment (AlisaVlasova) [reply src/index.html:37]: you should have 2 BEM blocks card and stars
and keep styles for them in two separate files as well

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,137 @@
+/* #region reset */
```
Comment (AlisaVlasova) [inline src/style.css:1]: these comments are redundant, just move styles to separate files

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+:root {
+  --main-color: #060b35;
+  --botton-color: #00acdc;
+  --product-price-color: #616070;
+  --content-wight: 166px;
+}
+
+html {
+  box-sizing: border-box;
```
Comment (AlisaVlasova) [inline src/style.css:19]: move it to * selector

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

**Review by Viktor-Kost — APPROVED** [review]
Good job!

**Pull request closed without merging**